backend/server/routers/movies.py

```diff
@@ -1,15 +1,22 @@
-from fastapi import APIRouter, Depends
-from pydantic import BaseModel
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel, Field
+from typing import Optional
 from ..util import DB, newId
 from ..dependencies import auth, admin
 
 router = APIRouter()
 
+# Temporary in-memory storage for reviews
+reviews_store = {}
 
 # ---------------------------------------------------------------------------- #
 #                                    Models                                    #
 # ---------------------------------------------------------------------------- #
-
+class CreateMovieReviewRequest(BaseModel):
+    movie_id: str
+    rating: int = Field(..., ge=1, le=5, description="Rating from 1 to 5")
+    content: Optional[str] = Field(None, max_length=250, description="Optional content about the movie")
```

Comment:
The text of the review isn't optional. The frontend requires there to be text for the review to be valid.